### PR TITLE
Fix typo in SpatialModel.plot_interactive()

### PR DESCRIPTION
**Pull request by QRemy**
Correct a typo in the name of some plot_interactive methods definition.

### CODE REVIEW

**Review by AtreyeeS — DISMISSED** [review]
Thanks @QRemy ! 

I don't know how we plan to maintain the deprecated functions. Should they remain in the tests (and also testing that the correct warning is raised) till the next release?

**Review by adonath — DISMISSED** [review]
Ops, that's embarrassing...:-) But the fix looks correct!

**Review by registerrier — COMMENTED** [review]
Thanks @QRemy . This looks good.

We just have one issue with the deprecation. Do we apply it to a bug fix as well? 
In that case, I would add `v1.0.1` as the version.

**Comment by adonath** [comment]
@registerrier I think this is a corner case. It is a bug-fix but also an API change. I'm not sure what our policy in this case see e.g. https://github.com/astropy/astropy-APEs/blob/main/APE2.rst#releases-and-backwards-compatibility

File: gammapy/modeling/models/spatial.py [hunk 1]
```diff
@@ -1246,7 +1271,13 @@ def plot(self, ax=None, geom=None, **kwargs):
             geom = self.map.geom
         super().plot(ax=ax, geom=geom, **kwargs)
 
+    @deprecated("v1.1", alternative="plot_interactive")
```
Comment (QRemy) [inline gammapy/modeling/models/spatial.py:1274]:
```suggestion
    @deprecated("v1.0.1", alternative="plot_interactive")
```

File: gammapy/modeling/models/spatial.py [hunk 2]
```diff
@@ -301,6 +302,7 @@ def plot(self, ax=None, geom=None, **kwargs):
             )
         return m.plot(ax=ax, **kwargs)
 
+    @deprecated("v1.1", alternative="plot_interactive")
```
Comment (QRemy) [inline gammapy/modeling/models/spatial.py:305]:
```suggestion
    @deprecated("v1.0.1", alternative="plot_interactive")
```

**Review by registerrier — APPROVED** [review]
Thanks @QRemy . Looks good.

**Pull request merged**